sonaranalyzer-dotnet/its/expected/Ember-MM/Ember Media Manager-{9B57D3AB-AF12-4012-B945-284C2448DC81}-S1075.json

```diff
@@ -0,0 +1,147 @@
+{
+"issues":  [
+{
+"id":  "S1075",
+"message":  "Refactor your code not to use hardcoded absolute paths or URIs.",
+"location":  {
+"uri":  "sources\Ember-MM\Ember%20Media%20Manager\dlgAbout.vb",
```

Comment:
Pretty sure you checked them all but are they all real issues?

Reply:
Yep.